src/VirtualMonoRepo/InitializeVMR.proj

```diff
@@ -36,14 +26,11 @@
   <Target Name="InitializeCleanVmr">
     <RemoveDir Directories='$(VmrDir)' Condition=" EXISTS('$(VmrDir)') " />
     <MakeDir Directories="$(VmrDir)" />
-    <MakeDir Directories="$(VmrGitInfoDir)" Condition=" '$(VmrGitInfoDir)' != '' " />
     <MakeDir Directories="$(TmpDir)" Condition=" !EXISTS('$(TmpDir)') " />
-    <MakeDir Directories="$(VmrDir)/src" />
     <Copy SourceFiles="$(MSBuildThisFileDirectory)/source-mappings.json" DestinationFolder="$(VmrDir)/src" />
 
     <Exec WorkingDirectory="$(VmrDir)" Command="git init -b $(VmrDefaultBranch)" />
-    <Exec WorkingDirectory="$(VmrDir)" Command="git add -A" />
-    <Exec WorkingDirectory="$(VmrDir)" Command="git commit -m 'Initial commit of the VMR%0D%0A%0D%0A[[ commit created by automation ]]'" />
+    <Exec WorkingDirectory="$(VmrDir)" Command="git commit --allow-empty -m 'Initial commit of the VMR%0D%0A%0D%0A[[ commit created by automation ]]'" />
```

Comment:
Fair point. Think about it, it probably doesn't matter actually. Git always switches to CR in its files and this will be in the commit object file. So I guess we could just do CR here